paddle/fluid/ir_adaptor/translator/op_compat_info.h

```diff
@@ -83,8 +83,9 @@ class OpNameNormalizer {
       std::string data = "@GRAD";
 
       size_t first_grad_pos = arg_name.find_first_of(target);
-      std::string legacy_name =
-          this->GetLegacyArgName(op_type, arg_name.substr(0, first_grad_pos));
+      size_t type_pos = op_type.find(target);
```

Comment:
这里是不是这样改就行了
```suggestion
      size_t first_grad_pos = arg_name.find(target);
```
然后其他地方不动。
因为这里就是只把_grad前面的部分抽出来先映射一次。因为我之前对find_first_of的理解是错误的，我以为它可以找到第一个_grad，其实它找到的是_grad这五个字符中第一个有匹配的，这里的first_grad_pos应该就是有问题的，我理解把它改掉就好了